src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,12 @@
+import { TodoInfo } from '../TodoInfo';
+
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList = ({ todos }) => (
+  <>
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo todo={todo} key={todo.id} />
```

Comment:
There is no check for the existence of 'todos' before trying to map over it. If 'todos' is undefined, the application will crash. Always ensure the existence of an array before trying to map over it.

Reply:
not fixed
add some check if todos exist
https://github.com/mate-academy/react_static-list-of-todos-js/pull/132#discussion_r1271441846